ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,7 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- heating_water_temperature_setpoint:
```

Comment:
See above comment. Setpoint should not be on the tank itself. You should be able to use `supply_water_temperature_setpoint` or similar on the system instead.